apps/src/templates/studioHomepages/CourseBlocks.jsx

```diff
@@ -162,33 +162,44 @@ class CourseBlocksCsfLegacy extends Component {
   }
 }
 
-export class CourseBlocksHoc extends Component {
+export class CourseBlocks extends Component {
   static propTypes = {
-    isInternational: PropTypes.bool
+    tiles: PropTypes.arrayOf(PropTypes.string).isRequired
```

Comment:
I wonder if it's worth a comment explaining that these are not just any strings, but jQuery selectors.

Reply:
Done.